tests/test_orient_symmetric.py

```diff
@@ -122,7 +136,12 @@ def test_estimate_rotations(n_img, L, order, dtype):
         )
 
     # Assert mean angular distance is reasonable.
-    if order > 4:
+    if order == 2:
+        # For small n-img, we get some bad estimates so checking the
+        # mean gets poor results with some runs. Instead we'll check that
+        # we get high percentage of good estimates.
+        assert np.mean(ang_dist < 6) > 0.80
+    elif order > 4:
         assert np.mean(ang_dist) < 5
     else:
```

Comment:
Maybe rewrite this if so it becomes `if order == 2: ... elif order == 3 or order == 4: ... else: ...`?

Reply:
Done